### ISSUE TITLE
Phase 6A: Reduce user signing from 3 to 1 transactions

### ISSUE THREAD

**Issue by prayagd** [pendulum-chain/vortex#35]
## Context
Follow-up ticket to phase 5. Currently the user has to sign 3 different transactions on the phase 5 version. This ticket to club all them in one single transaction. 

_There are two tickets for phase 6, the other one is https://github.com/pendulum-chain/pendulum-pay/issues/36_

## Requirement
Similarly to our flow on Stellar, the prototype should create a Pendulum ephemeral account for the user.

## TODO 
* at the beginning of the flow create a new random keypair for Pendulum that will only be managed in the browser – this is the Pendulum ephemeral account (PEA)
* before we start to submit the first transaction (which right now is the Nabla approval transaction), create a batch transaction that sends
  * the specified USDT to the PEA
  * 1 native token to the PEA (to pay for transaction fees)
* let the user sign this transaction and submit it
* once successful, create three transactions for the PEA and sign them directly in the browser
  * Nabla approval
  * Nabla swap
  * Spacewalk
* once the Spacewalk transaction is completed, read the remaining account balances of the PEA (native, input and output tokens of the Nabla swap)
* create, sign and submit a final transaction for the PEA that sends these tokens back to the user account
  * implemented as a batch transaction
  * only add tokens that have a non-zero balance to the batch
  * for the native token: use `transfer_allow_death`

## Shortcomings and Outlook
This section is only relevant for later iterations.

The PEA needs to have some native tokens in order to pay for transaction fees. The solution of this ticket requires that these tokens are "donated" by the user itself. This might not work at later iterations when the offramping flow will start on a different chain as native tokens will not be available.

Alternatively, we can use a funding account to provide these tokens. However, there is no multisig solution comparable to Stellar that will ensure that the user can't defraud us of these funds.

It could be that a safe solution can only be achieved with some custom onchain logic ("automation pallet").
